calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/parameter.py

```diff
@@ -89,3 +89,14 @@ def parameters(self) -> dict:
                     p_dict[k] = v
 
             return p_dict
+
+    def get_parameters_values(self) -> dict:
+        """Return the values of the Parameter objects from this class as a dictionary."""
+        with self.lock:
+            p_dict = {}
```

Comment:
nit: this code is almost the same as the previous one.
Code can be reused and then extended with a comprehension:

e.g., `return {k: v.value for k,v in self.parameters()}`

Reply:
(it could be argued that the original implementation is also suited for a comprehension)

Reply:
Addressed in https://github.com/tier4/CalibrationTools/pull/208/commits/184701ce4a34037ce28b397f4e611692f11f35d2

Reply:
@amadeuszsz 
`parameters` already uses a lock so the one in `get_parameters_values` should not be needed.
We are using RLock so while it may not be needed, I think it is better to remove the second `with self.lock`

Reply:
Addressed in https://github.com/tier4/CalibrationTools/pull/208/commits/237330de94fb6dd773c9511270f002aac733b6ff